layout/_third-party/copy-code.swig

```diff
@@ -25,6 +25,7 @@
       {% endif %}
       ta.blur(); // For iOS
       $(this).blur();
+      document.body.removeChild(ta);
```

Comment:
For what that changes?

Reply:
If `font-size` is lower than 16px(default), it doesn't matter whether to remove this. But when I test it with a larger `font-size`(e.g. 24px), the `<textarea>` element will cover the screen and affect mouse events. So it has to be removed.